### PR TITLE
[MNG-7684] Mention `maven.config` parse change in 3.9.0 release notes

### PR DESCRIPTION
**Pull request by Stephan202**
See [MNG-7684](https://issues.apache.org/jira/browse/MNG-7684) for context.

### CODE REVIEW

File: content/markdown/docs/3.9.0/release-notes.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ If you have any questions, please consult:
 * Several backports from Maven 4.x line.
 * Long outstanding issue fixes from Maven 3.8.x line.
 * General fixes and improvements.
+* Each line in `.mvn/maven.config` is now interpreted as a single argument. That is, if the file contains multiple arguments, these must now be placed on separate lines.
```
Comment (Stephan202) [inline content/markdown/docs/3.9.0/release-notes.md:46]: I'm certainly open to other ways of phrasing this; just lemme know :)

**Review by rickie — APPROVED** [review]

**Comment by cstamas** [comment]
Will reuse it here instead, where I list almost all notable and breaking changes: https://github.com/apache/maven-site/pull/381

**Comment by cstamas** [comment]
Page updated https://maven.apache.org/docs/3.9.0/release-notes.html

**Comment by Stephan202** [comment]
Tnx!

**Pull request closed without merging**